client/Dockerfile

```diff
@@ -2,11 +2,14 @@ FROM grpc/cxx:1.12.0
 
 WORKDIR /data
 
+ENV FOOD_NAME="milk"
+ENV SERVICE_URL="localhost:50051"
+
 COPY client/client.cc ./client/
 COPY proto/foodfinder.proto ./proto/
 COPY Makefile ./
 
 RUN make food-finder-client && make clean
 
-CMD ["./food-finder-client"]
+CMD ["sh", "-c", "./food-finder-client $FOOD_NAME $SERVICE_URL"]
```

Comment:
If I want to query some food other than milk, do I need to build a new image? My interactive solution is to write a interactive command line interface and run locally.

Reply:
Nope, you can just pass in an env variable when running it from the command line, e.g. `docker run --rm -e "SERVICE_URL=104.196.112.54:50051" -e "FOOD_NAME=pepper" food-finder-client`.